privacyidea/lib/eventhandler/base.py

```diff
@@ -118,17 +125,29 @@ def conditions(cls):
         :return: dict
         """
         realms = get_realms()
+        resolvers = get_resolver_list()
         cond = {
-            "realm": {
+            CONDITION.REALM: {
                 "type": "str",
                 "desc": _("The user realm, for which this event should apply."),
                 "value": list(realms)
             },
-            "tokenrealm": {
+            CONDITION.RESOLVER: {
+                "type": "str",
+                "desc": _("The user resolver, for which this event should apply."),
+                "value": list(resolvers)
+            },
+            CONDITION.TOKENREALM: {
                 "type": "multi",
                 "desc": _("The token realm, for which this event should "
                           "apply."),
-                "value": [{"name": r} for r in realms.keys()]
+                "value": [{"name": r} for r in list(realms)]
+            },
+            CONDITION.TOKENRESOLVER: {
+                "type": "multi",
+                "desc": _("The token resolver, for which this event should "
+                          "apply."),
+                "value": [{"name": r} for r in list(resolvers)]
```

Comment:
1. just `for r in resolvers` is enough
2. is it intended that multiple (token) resolvers can be selected here but only one (user) resolver?

Reply:
yes. For whichever reason.

Reply:
done in 081876c1